content/api/commands/session.md

```diff
@@ -0,0 +1,729 @@
+      expect(response.status).to.eq(200)
+    })
+  })
+})
+
+it('is a redundant test', () => {
+  /* which you can now delete! */
+})
+```
+
+### Cross-domain sessions
+
+It's possible to switch domains while caching sessions, just be sure to
+explicitly visit the domain in your login command before calling `cy.session()`.
+
+```jsx
+const login = (name) => {
+  if (location.hostname !== 'example.com') {
+    cy.visit('example.com')
+  }
+  cy.session(name, () => {
+    cy.visit('/login')
+    // etc
+  }, {
+    validate() {
+      cy.request('/whoami', {
+        headers: { 'Authorization' : localStorage.token }
+        method: 'POST'
+      }).its('statusCode').should('equal', '200')
+    }
+  })
+}
+
+it('t1', ()=>{
+  login('bob')
+  // do things on example.com
+})
+
+it('t2', ()=>{
+  cy.visit('anotherexample.com')
+  // do things on anotherexample.com
+})
+
+it('t3', ()=>{
+  login('bob')
+  // do things on example.com
+})
+```
+
+## Notes
+
+### When the page and active session data are cleared
+
+The page is cleared and all active session data (cookies, `localStorage`, and
+`sessionStorage`) across all domains are cleared automatically when
+`cy.session()` runs. This guarantees consistent behavior whether a session is
+being created or restored and allows you to switch sessions without first having
+to explicitly log out.
+
+|                      |              Current page cleared               |           Active session data cleared           |
+| -------------------- | :---------------------------------------------: | :---------------------------------------------: |
+| Before `setup`       | <Icon name="check-circle" color="green"></Icon> | <Icon name="check-circle" color="green"></Icon> |
+| Before `validate`    | <Icon name="check-circle" color="green"></Icon> |                                                 |
+| After `cy.session()` | <Icon name="check-circle" color="green"></Icon> |                                                 |
+
+Because calling `cy.session()` clears the current page in addition to restoring
+cached session data, [`cy.visit()`](/api/commands/visit) must always be
+explicitly called afterwards to ensure a page is visited.
+
+### Session caching
+
+Once created, a session for a given `id` is cached for the duration of the spec
+file. You can't modify a stored session after it has been cached, but you can
+always create a new session with a different `id`.
+
+In order to reduce development time, when running the Test Runner in "open"
+mode, sessions will be cached _between spec file runs_ as long as the `setup`
+function hasn't changed.
+
+### Explicitly clearing sessions
+
+When running the Test Runner in "open" mode, you can explicitly clear all
+sessions and re-run the spec file by clicking the "Clear All Sessions" button in
+the [Instrument Panel](#The-Instrument-Panel).
+
+For debugging purposes, all sessions can be cleared with the
+`Cypress.session.clearAllSavedSessions()` method.
```

Comment:
This is ancillary, but `Cypress.session` also needs to be documented under the "Cypress API" section. Similar to how `Cypress.spec` is documented for example: https://deploy-preview-4004--cypress-docs.netlify.app/api/cypress-api/spec.

The `Cypress.session` doc page should also indicate this feature is still experimental and link to `cy.session` doc as well.

Reply:
Done in 8e499112dc0bbc4bfdd66014fc96eb3cf48e2863